valkey.conf

```diff
@@ -299,6 +299,14 @@ tcp-keepalive 300
 #
 # tls-session-cache-timeout 60
 
+# Trusted clients
+#
+# By default the list of trusted IP is empty and the client on host machine's
+# netmask and unix socket clients are being considered as trusted client.
+# Example to add the list of IPs to be considered trusted as follows.
+#
+# trusted-addresses 10.0.0.1 11.0.0.2 10.0.0.9
```

Comment:
Can we accept CIDR ranges too?

Reply:
yes, we have plan as the following steps:

step 1: trusted-addresses only accept the lists for individual ip addresses
step 2: accept unit socket as the trusted client (separated pr for this)
step 3:  allow CIDR range as arguments (another pr follow step 2)